continue-reading.html

```diff
@@ -2,12 +2,85 @@
 <html>
   <head>
     <title>Continue Reading</title>
+    <link href="c-r-style.css" rel="stylesheet"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Lexend+Giga&display=swap" rel="stylesheet"/>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
-  <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+  <body id="everything"> <!--the whole content of the page is one class to which I applied attributes in CSS-->
+    
+    <section id="header"> <!--title and header section-->
+        <div id="title">Continue Reading</div>
+        <div id="take">or How To Have Something To Do in the Summer<hr><hr></div>
+    </section>
+
+    <div id="body">  <!--body of the page-->
+        <div id="main"></div>
+        <div id="sidebar">  
+          <a href="blog-post.html"><button id="date">July 30</button></a>
+          <h4>Just Another Post</h4>
+          <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+          <a href="blog-post.html"><button id="date">July 28</button></a>
+          <h4>And Another Post</h4>
+          <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis<hr>
+          <a href="blog-post.html"><button id="date">July 24</button></a>
+          <h4>One More Post</h4>
+          <section>lorem psum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus<hr>
+          <h4>Something of Note</h4>
+          <img src="https://i.imgur.com/uxJLfFs.png" alt="Mount Crested Butte" class="side";/>
+           Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+          <a href="learn-more.html"><br><button id="more">Learn More</button></a><hr>
+          <h4>Something of Less Note</h4>
+          <img src="https://i.imgur.com/SMwTgQ5.png" alt="Black Canyon of the Gunnison" class="side";/>
+          Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates 
+          <br><br>
+          <img src="https://i.imgur.com/bLtlF3Q.png" alt="Mountains and Trees in Alma, CO" class="side";/>
+          Quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis. 
+          <a href="learn-more.html"><br><button id="more">Learn More</button></a>
+      </div>

```

Comment:
If we add a little padding or margin on the left hand side of this, we can give a little space between our right and left columns and make it a little more user friendly